From 351876934e4b22f4a3a71936b01f2513e5bc50d7 Mon Sep 17 00:00:00 2001
From: Caio Oliveira <caiooliveirafarias0@gmail.com>
Date: Tue, 26 Jul 2016 12:50:54 -0300
Subject: [PATCH] mkcompile_h: It's builded under TravisCI [DO NOT MERGE]

Signed-off-by: Caio Oliveira <caiooliveirafarias0@gmail.com>
---
 scripts/mkcompile_h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/mkcompile_h b/scripts/mkcompile_h
index eb6b77f..913dc23 100755
--- a/scripts/mkcompile_h
+++ b/scripts/mkcompile_h
@@ -73,7 +73,7 @@ UTS_TRUNCATE="cut -b -$UTS_LEN"
 
   echo \#define UTS_VERSION \"`echo $UTS_VERSION | $UTS_TRUNCATE`\"
 
-  echo \#define LINUX_COMPILE_BY \"`echo $LINUX_COMPILE_BY | $UTS_TRUNCATE`\"
+  echo \#define LINUX_COMPILE_BY \"TravisCI\"
   echo \#define LINUX_COMPILE_HOST \"TeamVee-Kanas\"
 
   echo \#define LINUX_COMPILER \"`$CC -v 2>&1 | tail -n 1`\"
-- 
1.9.1

